src/db/models/workflowStatusOption.ts

```diff
@@ -0,0 +1,15 @@
+import * as t from 'io-ts';
+
+import { brandedType } from '../../util/io-ts';
+import type { Brand } from '../../util/types';
+
+export type WorkflowStatusOptionId = Brand<
+  number,
+  { readonly s: unique symbol },
+  'workflowStatusOption.id'
+>;
+
+export const WORKFLOW_STATUS_OPTION_ID = brandedType<
+  number,
+  WorkflowStatusOptionId
+>(t.number);
```

Comment:
Why didn't we added remaining fields planId,type and value in this model ?

Reply:
Because I'm taking minimalistic approach here :)

I only need `projectVersion` and `projectVersionPlan` defined, but need these ID properties defined in order to reference them. The rest will be defined when there's a need. It's a usual approach we take in this repo, look at `attachmentPrototype.ts`